assets/chezmoi.io/docs/reference/commands/cat.md

```diff
@@ -2,7 +2,7 @@
 
 Write the target contents of *target*s to stdout. *target*s must be files,
 scripts, or symlinks. For files, the target file contents are written. For
-scripts, the script's contents are written. For symlinks, the target target is
+scripts, the script's contents are written. For symlinks, the target path is
```

Comment:
I'm a little unsure about the wording of this one.

Reply:
+1 agree with @halostatue